spec/linkedca/majordomo.proto

```diff
@@ -268,3 +278,30 @@ message GetSSHCertificateStatusResponse {
 	RevocationReasonCode reason_code = 3;
 	google.protobuf.Timestamp revoked_at = 4;
 }
+
+message GetACMEAccountRequest {
+	oneof identifier {
+		string id = 1;
+		string kid = 2;
+	}
+}
+
+message CreateACMEAccountRequest {
+	string preferred_id = 1;
+	bytes jwk = 2;
+    ACMEAccount.Status status = 3;
+    repeated string contact = 4;
+    bool terms_of_service_agreed = 5;
+}
```

Comment:
Should there be a property to link this to a specific provisioner? I remember we used to allow an ACME account to be used with any ACME provisioner in the past, but I believe we changed that at some point?

Reply:
So the goal is to have linked authority X sync ACME accounts from provisioner Y to `majordomo`, so that linked authority Z get the accounts too, and for those to be active for all provisioners? And those authorities are totally independent; not clustered?